docs/DSA-Problem-Solution/partition_equal_subset_sum.md

```diff
@@ -0,0 +1,54 @@
+# Partition Equal Subset Sum Algorithm (LeetCode #416)
```

Comment:
We need on top

```md
---
id: ___
title: ___
sidebar_label: ___
sidebar_position: ___
description: ___
tags: [___, __, ]
---
```

### Detailed Guidelines

- id: Unique identifier for the route.
- title: Title of the problem.
- sidebar_label: Problem number or label for the sidebar.
- tags: Relevant tags for the problem.
- description: A short SEO-friendly description. No HTML tags.